loopd/swapclient_server.go

```diff
@@ -991,6 +991,19 @@ func (s *swapClientServer) GetLsatTokens(ctx context.Context,
 	return s.GetL402Tokens(ctx, req)
 }
 
+// FetchL402Token fetches a L402 Token from the server. This is required to
```

Comment:
What happens if it already has a L402 token? Shouldn't it fail in this case?
Maybe add `--force` flag to fetch an additional token, if there is already a token.
I propose to add these details to the comment.